src/utils/actions/writeAction.ts

```diff
@@ -1,24 +1,65 @@
-import { mutate } from '~amplifyClient';
-import { ContractEvent } from '~types';
+import { Extension, getExtensionHash } from '@colony/colony-js';
+import { mutate, query } from '~amplifyClient';
+import { ColonyActionType, ContractEvent } from '~types';
 import { verbose } from '~utils';
 
+interface ActionFields extends Record<string, any> {
+  initiatorAddress: string;
```

Comment:
Now that the initiatorAddress is needed to determine whether to show in the actions list, making it explicit here for type safety